docs/DeveloperGuide.md

```diff
@@ -659,86 +669,170 @@ Given below are instructions to test the app manually.
+        Trying the same command again on the same person will still show that
+        the tags have been added but only unique tags are shown.
+    3. Test case: `add-t 1029 r/friends`<br>
+        Expected: No `Person` is edited. Error showing invalid index shown
+        in the status bar. Assuming there are less than 1029 `Persons` in the list.
+    4. Test case `add-t`, `add-t 10`<br>
+        Expected: No `Person` is edited. Error showing invalid command format shown in the status bar.
 
-2. _{ more test cases …​ }_
 
 [↑ Back to top of section](#7-instructions-for-manual-testing)
 
 [↑ Back to table of contents](#table-of-contents)
 
-<div style="page-break-after: always"></div>
+## 7.5 Deleting tags from a contact:
+1. Deleting tags from a `Person`:
+    1. Prerequisites: List all `Persons` using the `list` command. Multiple `Persons` in the list. Assuming the first
+        `Person` in the list has the remark tag `friends` and module tag `CS2103T`.
+    2. Test case: `delete-t 1 r/1`<br>
+        Expected: The first `Person` in the list is edited to have the remark tag `friends` removed. 
+    3. Test case: `delete-t 1 r/1 m/1`<br>
+        Expected: The first `Person` in the list is edited to have the remark tag `friends` and module tag `CS2103T` removed.
+    4. Test case: `delete-t 1 r/10`<br>
+        Expected: No `Person` is edited. Error showing invalid index shown
+        in the status bar. Assuming there are less than 10 remark tags in the first `Person`'s remark tag list.
 
-## 7.4 Saving data
+[↑ Back to top of section](#7-instructions-for-manual-testing)
 
-1. Dealing with missing/corrupted data files
+[↑ Back to table of contents](#table-of-contents)
 
-    1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
+## 7.6 Searching for a contact:
+1. Searching for a `Person`:
+    1. Prerequisites: List all `Persons` using the `list` command. Multiple `Persons` in the list. 
+    2. Test case: `search cs`<br>.
+        Expected: All `Persons` whose information contains `cs` in any of the fields (name, email, address, telegram etc.).
+        For instance, if a person with a cca tag `ICS` would be shown in the list.
+    3. Test case: `search alex may`<br>
+        Expected: All `Persons` whose information contains `alex` and `may` in any of the fields (name, email, address, telegram etc.).
+        For instance, if a person with a name `Alex May` would be shown in the list.
+    4. Test case: `search r/friends`<br>
+        Expected: All `Persons` whose information contains `friends` in any of the remark tags.
+        For instance, if a person with a remark tag `friends` would be shown in the list.
+    
+
+[↑ Back to top of section](#7-instructions-for-manual-testing)  
 
-2. _{ more test cases …​ }_
+[↑ Back to table of contents](#table-of-contents)
+
+## 7.7 Opening a contact's social media links in app:
```

Comment:
Ditto, person vs contact. Also probably good to keep the spacing between the heading and the content consistent.

e.g. glossary has spacing between heading of letter and content. this section should have spacing as well maybe?